packages/composables/src/getters/cartGetters.ts

```diff
@@ -3,6 +3,8 @@ import { Cart, LineItem } from '@vue-storefront/spree-api/src/types';
 
 export const getCartItems = (cart: Cart): LineItem[] => cart?.lineItems || [];
 
+export const getCartItemId = (lineItem: LineItem): number => lineItem?._variantId;
```

Comment:
The name `getCartItemId` may be confusing, since line items have their own id as well. Maybe `getCartItemVariantId`, just to be explicit?

Reply:
Fixed it.